SIT.Manager/ViewModels/Play/DirectConnectViewModel.cs

```diff
@@ -75,6 +75,28 @@ public partial class DirectConnectViewModel : ObservableRecipient
         QuickPlayCommand = new AsyncRelayCommand(async () => await ConnectToServer(true));
     }
 
+    public Task ConnectToServer(string address, string username, string password) 
+    {
+        if(!string.IsNullOrEmpty(address))
+        {
+            LastServer = address;
+        }
```

Comment:
If the address is null you'll need to set it to 127.0.0.1 otherwise it'll fail the address validation rule and never start the local server